langchain4j-open-ai/src/test/java/dev/langchain4j/model/openai/OpenAiEmbeddingModelIT.java

```diff
@@ -71,6 +73,32 @@ void should_embed_multiple_segments() {
         assertThat(response.finishReason()).isNull();
     }
 
+    @Test
+    void should_embed_multiple_batch_segments() {
+
+        // given
+        List<TextSegment> segments = Stream.generate(() -> TextSegment.from("hello world"))
+                .limit(50)
```

Comment:
nit: Could you please create a specific `OpenAiEmbeddingModel` inside this test, otherwise it is not obvious how 50 relates to 10